### PR TITLE
Show message when trying to enable or disable the crash reporter

### PR DESCRIPTION
**Pull request by KrauseFx**
Since people might use it in their build script or similar

Fixes https://github.com/fastlane/fastlane/issues/4800

### CODE REVIEW

File: fastlane/lib/fastlane/commands_generator.rb [hunk 1]
```diff
@@ -171,6 +187,12 @@ def ensure_fastfile
       return false
     end
 
+    def show_crashreporter_note
+      UI.important("fastlane doesn't use a crash reporter any more")
+      UI.important("Instead please submit an issue on GitHub: https://github.com/fastlane/fastlane")
```
Comment (mfurtak) [inline fastlane/lib/fastlane/commands_generator.rb]: Link directly to issues? `https://github.com/fastlane/fastlane/issues`

**Comment by mfurtak** [comment]
1 small suggestion, otherwise LGTM

**Comment by KrauseFx** [comment]
Alright updated. Did you mean `/issues` or `/issues/new`? I think `/issues` is better, since then people might look for existing ones before creating a new one

**Comment by mfurtak** [comment]
Yup, that looks good! 🚢

**Pull request merged**